R/utils-assert.R

```diff
@@ -66,11 +66,19 @@ assert_proportion <- function(x, name = deparse(substitute(x))) {
   }
   invisible(x)
 }
-  
+
   assert_relatives <- function(x, name = deparse(substitute(x))) {
     if (x[[1]] != 1) {
       stop(sprintf("'%s[[1]]' must be 1", name), call. = FALSE)
     }
     assert_non_negative(x, name)
     invisible(x)
  }
+
+  assert_01 <- function(x, name = deparse(substitute(x))) {
```

Comment:
How about accepting logical too?

Reply:
Does odin accept logicals? I guess as.integer works on logicals too

Reply:
yes, that was my thought. you'll still pass 0/1 to odin, but at the R level could accept FALSE/TRUE and run them through as.integer

Reply:
Updated in https://github.com/mrc-ide/sircovid/pull/272